src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -530,11 +536,32 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
      * @return {void}
      */
     updateReplies = (id: string, replies: Array<Comment>) => {
-        const { api, file } = this.props;
+        const { activeFeedEntryId, api, file, history } = this.props;
+        const { feedItems } = this.state;
 
-        const feedAPI = api.getFeedAPI(false);
+        if (!feedItems) {
+            return;
+        }
 
+        const feedAPI = api.getFeedAPI(false);
         feedAPI.file = file;
+
+        // Detect if replies are being hidden and activeFeedEntryId belongs to a reply
+        // that is in currently being updated parent, in order to disable active item
+        if (
+            activeFeedEntryId &&
+            replies.length === 1 &&
+            feedItems.find(
```

Comment:
Should this be `.some`? It looks we do a form of this check in quite a few places. Would it be meaningful to move it to a utility function?

Reply:
Right, I've changed to `some` and also noticed that I could actually use a method (`getCommentFeedItemByReplyId`) I've created earlier, that already has logic to find the parent based on reply id and have used it now.
I've merged this PR because of time constraints but opened a new one for these minor fixes: https://github.com/box/box-ui-elements/pull/3148